django/db/migrations/operations/models.py

```diff
@@ -216,6 +216,27 @@ def database_forwards(self, app_label, schema_editor, from_state, to_state):
                     related_object.field,
                     to_field,
                 )
+            # Rename M2M fields whose name is based on this model's name.
+            fields = zip(old_model._meta.local_many_to_many, new_model._meta.local_many_to_many)
```

Comment:
I copied that pattern from 41b337ef, could we table that improvement to a separate ticket?